modules/admin_manual/pages/configuration/files/external_storage/s3_compatible_object_storage_as_primary.adoc

```diff
@@ -109,6 +116,8 @@ $CONFIG = [
         'arguments' => [
             // replace with your bucket
             'bucket' => 'owncloud',
+            // uncomment to enable server side encryption
+            //'serversideencryption' => 'AES256',
```

Comment:
Afaik server side encryption on s3 is not supported??

Reply:
@mmattel these comment lines do seem odd???